<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_campaign</class>
 <widget class="QWidget" name="form_campaign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_bug">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[New/Edit campaign]</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_id" stretch="0,0,0,50">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_id">
       <property name="text">
        <string>Id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_id">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Identification]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_description">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_description">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Description]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_system">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>200</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_system">
       <property name="text">
        <string>System:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_system">
       <property name="placeholderText">
        <string>[System]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_version">
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_version">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Version]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_blocks">
       <property name="text">
        <string>Test blocks:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_blocks">
       <property name="placeholderText">
        <string>[Blocks]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_creation">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>300</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_creation">
       <property name="text">
        <string>Creation time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_creation">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Creation time]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_update">
       <property name="text">
        <string>Last update:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_update">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Last update]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_campaign">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>420</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_create_campaign">
       <property name="text">
        <string>Create campaign</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
